### PR TITLE
GitHub actions for running tests in CI/CD pipeline

### PR DESCRIPTION
**Pull request by ianco**
Added workflow for github action (initially set to run on "push")

Updated manage script to provide support for running in CI/CD pipeline

Runs for acapy/acapy, manage script waits for indy ledger and tails server, workflow fails if any tests fail.

PR is "WIP" for now, working on adding support for .net/.net and acapy/.net tests

### CODE REVIEW

File: manage [hunk 1]
```diff
@@ -10,12 +10,18 @@ export DOCKERHOST=${APPLICATION_URL-$(docker run --rm --net=host eclipse/che-ip)
...
+export LEDGER_TIMEOUT=60
 # Running on Windows?
 if [[ "$OSTYPE" == "msys" ]]; then
   # Prefix interactive terminal commands ...
   terminalEmu="winpty"
 fi
 
+export INTERACTIVE="-it"
+if [[ "$NO_TTY" == "1" ]]; then
+  INTERACTIVE=""
```
Comment (WadeBarnes) [inline manage]:
```suggestion
  unset INTERACTIVE
```

**Review by WadeBarnes — DISMISSED** [review]

**Comment by WadeBarnes** [comment]
One minor suggestion, otherwise looks good.

**Comment by ianco** [comment]
Added workflows to run acapy/acapy, acapy/dotnet and dotnet/dotnet

@nodlesh  how to specify to pull code from the acapy and dotnet master branches rather than the latest published version?

**Comment by nodlesh** [comment]
@ianco simply tag -master on the end of the agent types on the command line.
eg. 
./manage build -a acapy-master
./manage run -d acapy-master -t @AcceptanceTest -t ~@wip

**Comment by ianco** [comment]
Updated to run against acapy and dotnet master branch

Updated to run workflows on a schedule once per day (not on push)

**Comment by ianco** [comment]
DCO got screwed up

**Pull request closed without merging**